lib/globalconstraintregistrar.js

```diff
@@ -30,6 +30,91 @@ export class GlobalConstraintRegistrar extends ExtendTruffleContract(SolidityGlo
     return new this(contract);
   }
 
+  async proposeToAddModifyGlobalConstraint(opts={}) {
+    const defaults = {
```

Comment:
why are we using this model of a config object instead of parameters to the function if all the options are going to be undefined and throw an error if they don't exist? did we decide at some point that we wanted to use options instead of specific parameters for all functions in arc?

Reply:
I was continuing with the pre-existing pattern.  But I do prefer to use an options object like that.  I find it tends to be easier to make changes to an API while maintaining backwards compatibility.